python/paddle/fluid/tests/unittests/test_while_loop_op.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import print_function
+
+import numpy as np
+import unittest
+
+import paddle.fluid as fluid
+import paddle.fluid.core as core
+import paddle.fluid.layers as layers
+import paddle.fluid.framework as framework
+from paddle.fluid.executor import Executor
+from paddle.fluid.framework import Program, program_guard
+
+
+class TestWhileLoop(unittest.TestCase):
+    def test_simple_net(self):
+        def cond(i):
+            return layers.less_than(i, ten)
+
+        def body(i):
+            return layers.increment(i)
```

Comment:
Can you try whether you can avoid using increment in these tests?

Reply:
"layers.elementwise_add" has been used instead of "layers.increment"